report_async/models/report_async.py

```diff
@@ -71,7 +70,7 @@ class ReportAsync(models.Model):
         help="List all files created by this report background process",
     )
 
-    schedule_time = fields.Char(string='Schedule time')
+    schedule_time = fields.Datetime(string='Schedule Time')
```

Comment:
This would need migration scripts. Can we adapt the code in a way to avoid this change?